File: 백준/Bronze/2439. 별 찍기 － 2/별 찍기 － 2.java
import java.io.IOException;
import java.io.InputStreamReader;
import java.io.BufferedReader;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        
        int N = Integer.parseInt(br.readLine());
        StringBuilder result = new StringBuilder();
        
        for(int i = 1; i <= N; i++){
            result.append(" ".repeat(N-i) + "*".repeat(i));
            if(i < N){result.append("\n");}
        }
        
        System.out.print(result.toString());
    }
}

File: 백준/Bronze/2439. 별 찍기 － 2/별 찍기 － 2.kt
import java.io.BufferedReader
import java.io.InputStreamReader

fun main() {
    val br = BufferedReader(InputStreamReader(System.`in`))
    
    val N = Integer.parseInt(br.readLine())
    
    val result = StringBuilder()
    
    for(i in 1..N){
        result.append("*".repeat(i).padStart(N, ' '))
        if(i<N){result.append("\n")}
    }
    
    print(result.toString())
}